core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -0,0 +1,786 @@
+      String policyName,
+      String groupShortName,
+      String groupLongName,
+      @Nullable String parentGroupShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Drops a group that has the given short name for the given policy.
+   *
+   * @param policyName the policy name
+   * @param groupShortName the short name of the group
+   * @throws ExecutionException if the operation fails
+   */
+  default void dropGroup(String policyName, String groupShortName) throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Retrieves a group that has the given short name for the given policy.
+   *
+   * @param policyName the policy name
+   * @param groupShortName the short name of the group
+   * @return the group
+   * @throws ExecutionException if the operation fails
+   */
+  default Optional<Group> getGroup(String policyName, String groupShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Retrieves all groups for the given policy.
+   *
+   * @param policyName the policy name
+   * @return the groups
+   * @throws ExecutionException if the operation fails
+   */
+  default List<Group> getGroups(String policyName) throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Sets the given levels of the given policy to a user that has the given username.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param levelShortName the short name of the level to set
+   * @param defaultLevelShortName the short name of the default level. If null, the {@code
+   *     levelShortName} will be used as the default level
+   * @param rowLevelShortName the short name of the row level. If null, the {@code
+   *     defaultLevelShortName} will be used as the row level
+   * @throws ExecutionException if the operation fails
+   */
+  default void setLevelsToUser(
+      String policyName,
+      String username,
+      String levelShortName,
+      @Nullable String defaultLevelShortName,
+      @Nullable String rowLevelShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Adds the given compartment of the given policy to a user that has the given username. Before
+   * adding the compartment, levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param compartmentShortName the short name of the compartment
+   * @param accessMode the access mode
+   * @param defaultCompartment whether the compartment is the default compartment
+   * @param rowCompartment whether the compartment is the row compartment
```

Comment:
If my understanding is correct, several compartments can be assigned to the user. 
Using the word `the` implies this group will be override any group and be the only `defaultCompartment`, using the word `a` seems more correct as the group will be added as one of the `defaultCompartment`.
Likewise for rowCompartment. 
```suggestion
   * @param defaultCompartment whether the compartment is a default compartment
   * @param rowCompartment whether the compartment is a row compartment
```

What do you think ?

Reply:
> If my understanding is correct, several compartments can be assigned to the user

@Torch3333 Your understanding is correct. Let me update it as you suggested. Thanks!